src/main/java/io/jenkins/plugins/jfrog/CliEnvConfigurator.java

```diff
@@ -79,4 +79,11 @@ private static void excludeProxyEnvFromPublishing(EnvVars env) {
         String jfrogCliEnvExclude = env.getOrDefault(JFROG_CLI_ENV_EXCLUDE, JFROG_CLI_DEFAULT_EXCLUSIONS);
         env.put(JFROG_CLI_ENV_EXCLUDE, String.join(";", jfrogCliEnvExclude, HTTP_PROXY_ENV, HTTPS_PROXY_ENV));
     }
+
+    static String noProxyExtractor(String noProxyList) {
```

Comment:
"Extractor" means an object (class) that extracts things, but this is only a function.
Please consider this:
```suggestion
    static String createNoProxyValue(String noProxyList) {
```

If you agree with this change, please update the name of the test.